cadasta/plugin.py

```diff
@@ -201,26 +192,51 @@ def _create_options_wizard_action(self):
             callback=self.show_options_wizard
         )
 
-    def show_options_wizard(self):
-        """Show the options wizard."""
-        dialog = CadastaLogin()
-        dialog.show()
-        dialog.exec_()
+    def _create_project_creation_wizard(self):
+        """Create action for project creation wizard."""
+        icon_path = ':/plugins/cadasta-qgis-plugin/icon.png'
+        self.project_creation_wizard = self.add_action(
+            icon_path,
+            text=self.tr(u'Create Project'),
+            parent=self.iface.mainWindow(),
+            add_to_toolbar=False,
+            enabled_flag=True,
+            callback=self.show_project_creation_wizard
+        )
 
-    def _create_project_download_wizard_action(self):
+    def _create_project_download_wizard(self):
         """Create action for project download wizard."""
         icon_path = ':/plugins/cadasta-qgis-plugin/icon.png'
```

Comment:
hi @Gustry , it is fixed in another PR: https://github.com/kartoza/cadasta-qgis-plugin/pull/52